xcrypto/hash.go

```diff
@@ -24,16 +24,16 @@ const (
 )
 
 var (
-	MD5BlockSize    = int(cryptokit.HashBlockSize(md5))
-	MD5Size         = int(cryptokit.HashSize(md5))
-	SHA1BlockSize   = int(cryptokit.HashBlockSize(sha1))
-	SHA1Size        = int(cryptokit.HashSize(sha1))
-	SHA256BlockSize = int(cryptokit.HashBlockSize(sha256))
-	SHA256Size      = int(cryptokit.HashSize(sha256))
-	SHA384BlockSize = int(cryptokit.HashBlockSize(sha384))
-	SHA384Size      = int(cryptokit.HashSize(sha384))
-	SHA512BlockSize = int(cryptokit.HashBlockSize(sha512))
-	SHA512Size      = int(cryptokit.HashSize(sha512))
+	mD5BlockSize    = int(cryptokit.HashBlockSize(md5))
```

Comment:
acronyms capitalization is all or nothing. `md5BlockSize` won't hurt my eyes that much. Same for other symbols in this file.